docs/source/administrator/upgrading/upgrade-1-to-2.md

```diff
@@ -67,6 +70,20 @@ in the changelog for more information on other upgraded hub components.
 The default singleuser server is [JupyterLab](https://jupyterlab.readthedocs.io/), running on [Jupyter server](https://jupyter-server.readthedocs.io/en/latest/).
 To switch back to Jupyter Notebook either configure/rebuild your singleuser image to default to notebook, or see [the documentation on user interfaces](user-interfaces)
 
+## KubeSpawner disallows root users by default
+
+KubeSpawner will prevent processes executing as root in the singleuser container by default.
+You must set `singleuser.allowPrivilegeEscalation: true` to enable root users.
+For example, if you are using a [docker-stacks](https://jupyter-docker-stacks.readthedocs.io/) image and want to enable sudo:
+
+```yaml
+singleuser:
+  allowPrivilegeEscalation: true
+  uid: 0
+  extraEnv:
```

Comment:
This is the default CMD for a docker-stacks Dockerfile, and without it, `GRANT_SUDO` doesn't work. Since this will work no matter if our default `cmd` is unset or explicit, I suggest we be explicit about this. Whoever use the docker-stacks will then better grasp that GRANT_SUDO is an env related to their script as well, making people debug possible issues better overall I think.

```suggestion
  cmd: start-notebook.sh
  extraEnv:
```

Reply:
Good point! I've made it generic instead of being specific to docker-stacks. However it might be probably worth adding the `GRANT_SUDO` as a full docker-stacks example somewhere in the main docs?

Reply:
Yepp, very reasonable call as well!